src/components/security_manager/test/ssl_context_test.cc

```diff
@@ -199,6 +211,9 @@ class SSLTest : public testing::Test {
 
   static std::string client_certificate_data_base64_;
   static std::string server_certificate_data_base64_;
+
+  const std::vector<int> forced_unprotected_service;
```

Comment:
@ValeriiMalkov same as above

Reply:
@AByzhynar fixed at the commit 0eee356873e4c810e70e48657dc1f4dc1a8467fe